react/react1/week1/todo-list/src/componate/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import TodoList from './TodoList';
+import Header from './Header';
+import Footer from './Footer';
+import todoList from '../Todo';
+
+function createTodo(todoList) {
+    return (
+      <TodoList
+        key={todoList.id}
+        description={todoList.description}
```

Comment:
the props aren't relevant here, you can access them inside the component

Reply:
"For each item render a description and a deadline date. Before you start draw a mockup and identify the components with colours. You have to use more than two components. Think which props the components should take." 

I am confuse with assignment. Do I have to use CSS with components and Do I to create CSS file in  scr folder with index.js file? I have created separate CSS file in public folder.

Reply:
The assignment asked you to draw a mock-up before you started to code because that would help you visualize what you should do. It's nothing related to CSS, but you could style your app in App.css